QModManager/API/SMLHelper/Assets/PDAItem.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace QModManager.API.SMLHelper.Assets
+{
+    using Crafting;
+    using Handlers;
+
+    /// <summary>
+    /// A <see cref="Spawnable"/> item that appears in the PDA blueprints.
+    /// </summary>
+    /// <seealso cref="Spawnable" />
+    public abstract class PDAItem : Spawnable
+    {
+        /// <summary>
+        /// Override to set the <see cref="TechType"/> that must first be scanned or picked up to unlock the blueprint for this item. <para/>
+        /// If not overriden, it this item will be unlocked from the start of the game.
+        /// </summary>
+        public virtual TechType RequiredForUnlock => TechType.None;
```

Comment:
As a later revision, I plan on adding a new property "`virtual IEnumerable<TechType> UnlockedBy => null;`"  
This will accept any list of TechTypes such that when _any_ of those TechTypes are scanned, the `PDAItem` here is unlocked.

This will likely go hand in hand with normalizing the unlocking method that @kylinator25 came up with for attaching blueprint unlocks to Encyclopedia entries rather than other blueprints.